alerts/google-gce/cpu-utilization-too-high.v1.json

```diff
@@ -0,0 +1,30 @@
+{
+  "displayName": "CPU Utilization",
```

Comment:
Please match displayName with the ones in metadata.yaml, same for other policies.

Reply:
Display names in json files now match metadata.yaml.